booking/src/main/java/org/dcsa/conformance/standards/booking/action/BookingAction.java

```diff
@@ -32,13 +33,27 @@ public BookingAction(
       String actionTitle,
       int expectedStatus) {
     super(sourcePartyName, targetPartyName, previousAction, actionTitle);
-    this.expectedStatus = expectedStatus;
+    this.expectedStatus = Set.of(expectedStatus);
     this.dspReference =
         previousAction == null
             ? new OverwritingReference<>(null, new DynamicScenarioParameters(ScenarioType.REGULAR, null, null, null, null,null, null))
             : new OverwritingReference<>(previousAction.dspReference, null);
   }
 
+  public BookingAction(
+    String sourcePartyName,
+    String targetPartyName,
+    BookingAction previousAction,
+    String actionTitle,
+    Set<Integer> expectedStatus) {
+    super(sourcePartyName, targetPartyName, previousAction, actionTitle);
+    this.expectedStatus = expectedStatus;
+    this.dspReference =
+      previousAction == null
+        ? new OverwritingReference<>(null, new DynamicScenarioParameters(ScenarioType.REGULAR, null, null, null, null,null, null))
+        : new OverwritingReference<>(previousAction.dspReference, null);
+  }
```

Comment:
I was just looking briefly at this PR and see 2 almost identical constructors. What is the reason of keeping 2? Can't we just only keep one, and make the callers just supplying a Set? (even if it is having only 1 value)

Btw, if you can use List as well, that's preferred above Set.

Reply:
Yes, I meant the `BookingAction` constructor indeed. There is only 1 or 2 usages of the single `expectedStatus` argument, so I don't see the need of keeping multiple constructor for every variance of constructing this object. The calling method can just easily be adapted to supplying a Set with a single item. That avoids code duplication.